libnopegl/src/node_graphicconfig.c

```diff
@@ -35,29 +35,29 @@ struct graphicconfig_opts {
     struct ngl_node *child;
 
     int blend;
-    int blend_src_factor;
-    int blend_dst_factor;
-    int blend_src_factor_a;
-    int blend_dst_factor_a;
-    int blend_op;
-    int blend_op_a;
+    enum ngpu_blend_factor blend_src_factor;
```

Comment:
The unset value is -1 which is not associated with a corresponding enum. Isn't this a problem?

Reply:
It's not an issue, the enum type is just an int32_t/uint32_t (with fancy names). You can still compare them to int32_t/uint32_t.